<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>642</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>803</width>
    <height>642</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>803</width>
    <height>642</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>791</width>
      <height>641</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="logmenu">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>300</y>
        <width>531</width>
        <height>231</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>148</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="log_in">
           <property name="minimumSize">
            <size>
             <width>131</width>
             <height>71</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>131</width>
             <height>71</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>15</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>OpenHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>管理员登陆</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="set_admin">
           <property name="minimumSize">
            <size>
             <width>131</width>
             <height>71</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>131</width>
             <height>71</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>OpenHandCursor</cursorShape>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>管理员注册</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exit">
           <property name="minimumSize">
            <size>
             <width>131</width>
             <height>71</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>131</width>
             <height>71</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>13</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="text">
            <string>退出</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>188</width>
           <height>135</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="mainmenu">
     <property name="font">
      <font>
       <family>Arial</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <widget class="QPushButton" name="back">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="remov">
      <property name="geometry">
       <rect>
        <x>501</x>
        <y>394</y>
        <width>93</width>
        <height>35</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>下架商品</string>
      </property>
     </widget>
     <widget class="QPushButton" name="totalgood">
      <property name="geometry">
       <rect>
        <x>629</x>
        <y>394</y>
        <width>93</width>
        <height>35</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>库存盘点</string>
      </property>
     </widget>
     <widget class="QPushButton" name="findgood">
      <property name="geometry">
       <rect>
        <x>501</x>
        <y>318</y>
        <width>93</width>
        <height>35</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>商品查询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="possesion">
      <property name="geometry">
       <rect>
        <x>629</x>
        <y>470</y>
        <width>93</width>
        <height>35</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>资产盘点</string>
      </property>
     </widget>
     <widget class="QPushButton" name="changemes">
      <property name="geometry">
       <rect>
        <x>629</x>
        <y>318</y>
        <width>93</width>
        <height>35</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>修改信息</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ingood">
      <property name="geometry">
       <rect>
        <x>501</x>
        <y>242</y>
        <width>93</width>
        <height>35</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>商品入库</string>
      </property>
     </widget>
     <widget class="QPushButton" name="outgood">
      <property name="geometry">
       <rect>
        <x>629</x>
        <y>242</y>
        <width>93</width>
        <height>35</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>产品出库</string>
      </property>
     </widget>
     <widget class="QPushButton" name="vip">
      <property name="geometry">
       <rect>
        <x>501</x>
        <y>470</y>
        <width>93</width>
        <height>35</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="text">
       <string>会员管理</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
